src/screens/ui/AppletBottomTabNavigator.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  createBottomTabNavigator,
+  BottomTabNavigationOptions,
+} from '@react-navigation/bottom-tabs';
+import { useTranslation } from 'react-i18next';
+
+import { AppletDetailsParamList, appletDetailsScreenOptions } from '../config';
+import { AboutScreen, AppletActivityScreen } from '../ui';
+
+const Tab = createBottomTabNavigator<AppletDetailsParamList>();
+
+const AppletBottomTabNavigator = () => {
+  const { t } = useTranslation();
+  return (
+    <Tab.Navigator
+      screenOptions={appletDetailsScreenOptions as BottomTabNavigationOptions}>
```

Comment:
Something went wring if you have to write "as BottomTabNavigationOptions"
Try creating appletDetailsScreenOptions using BottomTabNavigationOptions type